### PR TITLE
AF-1132/1134: Not clear that deployment fails if no Kie Server configured

### PR DESCRIPTION
**Pull request by tiagobento**
See:
- https://issues.jboss.org/browse/AF-1134
- https://issues.jboss.org/browse/AF-1132

### CODE REVIEW

File: kie-wb-common-screens/kie-wb-common-project-editor/kie-wb-common-project-editor-client/src/main/resources/org/kie/workbench/common/screens/projecteditor/client/resources/i18n/ProjectEditorConstants.properties [hunk 1]
```diff
@@ -167,3 +167,8 @@ StartContainer=Start container?
 NoServersAvailableForProvisioning=Unable to provision due to no active servers available
 ContainerIdAlreadyInUse=Container name is already in use, please choose a different name.
 ProjectPreferences=Project Preferences
+DeployFailed=Deploy to server template failed.
```
Comment (jhrcek) [inline kie-wb-common-screens/kie-wb-common-project-editor/kie-wb-common-project-editor-client/src/main/resources/org/kie/workbench/common/screens/projecteditor/client/resources/i18n/ProjectEditorConstants.properties]: @tiagobento please replace "server template" by "server configuration" - this applies primarily to visible UI labels-  you can leave variable names as they are.. See recent renaming requested by PM for details   [RHPAM-525](https://issues.jboss.org/browse/RHPAM-525)

**Comment by jhrcek** [comment]
I'd like to test this locally. Jenkins execute full downstream build

**Comment by jhrcek** [comment]
Repo build failing because of mismatch between property name / ProjectEditorConstants method name. Please fix it.

```
00:42:20 [INFO]                   [ERROR] No resource found for key 'DeploymentSkippedDueToNoServerTemplateConfigured'
00:42:20 [INFO] com.google.gwt.i18n.rebind.AbstractResource$MissingResourceException: No resource found for key 'DeploymentSkippedDueToNoServerTemplateConfigured'
```

File: kie-wb-common-screens/kie-wb-common-project-editor/kie-wb-common-project-editor-client/src/main/resources/org/kie/workbench/common/screens/projecteditor/client/resources/i18n/ProjectEditorConstants.properties (hunk 1 above)
Comment (tiagobento) [reply kie-wb-common-screens/kie-wb-common-project-editor/kie-wb-common-project-editor-client/src/main/resources/org/kie/workbench/common/screens/projecteditor/client/resources/i18n/ProjectEditorConstants.properties]: Changed. Thanks, @jhrcek!

**Pull request merged**